.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 2.6.2
```

Comment:
I know this is for `asdf` version manager, but can't the asdf ruby plugin not use the same `.ruby-version` standard as `rbenv`, `chruby`, and `rvm`?

Reply:
I don't think so, I tried searching this right now and didn't found a way. I don't like this either makes the root project folder kind of dirt with so many version manager files, but its good for obvious reasons, so we remove this file or keep it?

Reply:
I dont really have a preference, just thought that we could avoid it, but id rather `asdf` users have a positive experience setting up the app.

Reply:
Looks like there is a [configuration flag](https://github.com/asdf-vm/asdf-ruby#migrating-from-another-ruby-version-manager) you can set on your `.asdfrc`